test/helpers/application_helper_test.rb

```diff
@@ -114,4 +114,18 @@ def test_generate_link_for
         '<a class="btn" href="javascript:foo()">link</a>')
     end
   end
+
+  describe 'host details ui path generation' do
```

Comment:
Would you mind writing a test in the "normal" case as well, where the name is present?